<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogInserirProfessor</class>
 <widget class="QWidget" name="DialogInserirProfessor">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>239</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inserir Professores</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>511</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <item row="1" column="0">
     <widget class="QLabel" name="label_1">
      <property name="text">
       <string>Nome:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="input_nome_prof"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Data de Nascimento:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDateEdit" name="input_dn_prof">
      <property name="displayFormat">
       <string>dd/MM/yyyy</string>
      </property>
      <property name="calendarPopup">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Situação:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="combo_sit_prof"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Regime Jurídico:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QComboBox" name="combo_reg_jur_prof"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Sede:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="combo_sede_prof"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Categoria:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QComboBox" name="combo_cat_prof"/>
    </item>
    <item row="8" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="buttom_fechar">
        <property name="text">
         <string>&amp;Fechar</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="buttom_inserir">
        <property name="text">
         <string>&amp;Inserir</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Disciplina:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QComboBox" name="combo_disciplina"/>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>input_nome_prof</tabstop>
  <tabstop>input_dn_prof</tabstop>
  <tabstop>combo_sit_prof</tabstop>
  <tabstop>combo_reg_jur_prof</tabstop>
  <tabstop>combo_sede_prof</tabstop>
  <tabstop>combo_cat_prof</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttom_fechar</sender>
   <signal>released()</signal>
   <receiver>DialogInserirProfessor</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>407</x>
     <y>345</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
